aviatrix/resource_aviatrix_dcf_trustbundle.go

```diff
@@ -0,0 +1,125 @@
+package aviatrix
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/AviatrixSystems/terraform-provider-aviatrix/v3/goaviatrix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAviatrixDCFTrustBundle() *schema.Resource {
+	return &schema.Resource{
+		CreateWithoutTimeout: resourceAviatrixDCFTrustBundleCreate,
+		ReadWithoutTimeout:   resourceAviatrixDCFTrustBundleRead,
+		UpdateWithoutTimeout: resourceAviatrixDCFTrustBundleUpdate,
+		DeleteWithoutTimeout: resourceAviatrixDCFTrustBundleDelete,
+
+		Schema: map[string]*schema.Schema{
+			"display_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringIsNotWhiteSpace,
+				Description:  "Display name for the DCF trust bundle.",
+			},
+			"bundle_content": {
+				Type:         schema.TypeString,
```

Comment:
This is a collection of PEM strings, so should we use TypeList and set the Elem to string?